charts/nexus-iq/templates/persistentVolume.yaml

```diff
@@ -1,10 +1,11 @@
-{{- if .Values.persistence.volumeConfiguration -}}
+{{- if not .Values.statefulset.enabled }}
+{{- if .Values.persistence.pdName -}}
 apiVersion: v1
 kind: PersistentVolume
 metadata:
-  name: {{ template "iqserver.fullname" . }}-data
+  name: {{ .Values.persistence.pdName }}
   labels:
-    {{- include "iqserver.labels" . | nindent 4 }}
+{{ include "iqserver.labels" . | indent 4 }}
```

Comment:
can this be indented to appear as more of a child of "labels:"?

Reply:
> can this be indented to appear as more of a child of "labels:"? I realize the indent thing probably renders it the same no matter where this bit of template code exists.

I think we will need to address this more globally as we progress - for now it's only intent is to be consistent with existing NXRM implementation

Reply:
hmmm. i could argue that the "spirit" of the task is to also keep these files consistent with themselves. 💭

Reply:
it is frustrating that these  are the *new* charts and they're still not consistent when declaring the exact same concept. 😞 (we have stuff to fix!)